data/themes/darktable.css

```diff
@@ -633,6 +633,15 @@ overshoot.right
   font-weight: 500;
 }
 
+#section_label *
+{
+  padding: 0;
+  color: @section_label;
+  font-family: sans-serif;
+  font-weight: 500;
+  border: 0;
+}
+
```

Comment:
You set 2 times with those lines same thing to just avoid a not wanted bottom border. To simplify CSS, replace lines 627 to 643 by following lines:

```
#section_label *
{
  padding: 0;
  color: @section_label;
  font-family: sans-serif;
  font-weight: 500;
}

#section_label
{
  border-bottom: 0.035em solid  @section_label;
}
```

Reply:
Forget a line above:

```
#section_label,
#section_label *
{
  padding: 0;
  color: @section_label;
  font-family: sans-serif;
  font-weight: 500;
}

#section_label
{
  border-bottom: 0.035em solid  @section_label;
}
```

Reply:
That won't work with the way the pango font are created to display section in blend gui. We look for the `section_label` on the code. So we need the font information there too. I'll add comments.

Reply:
> That won't work with the way the pango font are created to display section in blend gui. We look for the `section_label` on the code. So we need the font information there too. I'll add comments.

I suppose you answer to my first comment which was a mistake by forgotten a line. The second code proposed just about a minute ago your comment is the same as yours but just optimize to not repeat 2 times some lines and with the forgotten #section_label only line!

Reply:
Comments added.

Reply:
> > That won't work with the way the pango font are created to display section in blend gui. We look for the `section_label` on the code. So we need the font information there too. I'll add comments.
> 
> I suppose you answer to my first comment which was a mistake by forgotten a line. The second code proposed just about a minute ago your comment is the same as yours but just optimize to not repeat 2 times some lines and with the forgotten #section_label only line! And this second code works great on my side.

It works great because you are using "font-weight: 500;" which is almost a nop. Try using 900 here and also you need to restart dt as the font is created once at startup in the bauhaus for efficiency reasons. So just changing this in the user's CSS section won't work for this section label. (look for the **drawn mask** label in blen GUI).

Reply:
Hmm, I think my first comment (and mistake) doesn't help to understand what I suggest. My CSS code change (the correct one) is just the exact same thing as yours except that it uses less lines. Of course, if we don't set font-weight and other setting on #section_label directly (like with my mistake in my first comment), we will have issue you cited. But with my second comment and so #section_label added, I can change dynamically and correctly without restarting darktable. Tested by changing to 900 as your example.

Your code:

```
#section_label
{
  padding: 0;
  color: @section_label;
  border-bottom: 0.035em solid  @section_label;
  font-family: sans-serif;
  font-weight: 500;
}

#section_label *
{
  padding: 0;
  color: @section_label;
  font-family: sans-serif;
  font-weight: 500;
  border: 0;
}
```

is just the same as:

```
#section_label,
#section_label *
{
  padding: 0;
  color: @section_label;
  font-family: sans-serif;
  font-weight: 500;
}

#section_label
{
  border-bottom: 0.035em solid  @section_label;
}
```

Reply:
Just an add as I just see what you means by draw mask line. i have the issue you cited with both your code and my proposal, but it's normal as my proposal is the same CSS code as yours, optimized.

Reply:
I see, that's not the code I tried.... I see that you have added a second corrected version that I didn't see...

Reply:
@Nilvus : works ! I'll merge then. Thanks for the review.

Reply:
My pleasure! Thanks for your work. All good now.

Reply:
Thanks @Nilvus , I did find a discepencies when using elegant-dark, the font in the label "paremetric mask" and "mask refinement" were not the same as the one in "drawn mask" label. Found by using a bigger fonts for testing. Now fixed the blend section labels are all identical and well readable !

Reply:
Good catch!